client/app/config/steps/password.coffee

```diff
@@ -2,4 +2,16 @@ module.exports = {
     name: 'password',
     route: 'password',
     view : 'steps/password'
+
+    # If OK, return null
+    # if not return an Array of errors
+    validate: (data) ->
+        return null
+
+    submit: (data) ->
+        jQuery = global.jQuery
+        jQuery ?= require 'jQuery'
```

Comment:
For my culture, why use two potentially different sources for the jQuery dependance here? Is it not a potential source of issues?